api-put-object.go

```diff
@@ -84,6 +85,9 @@ func (opts PutObjectOptions) Header() (header http.Header) {
 	if opts.StorageClass != "" {
 		header[amzStorageClass] = []string{opts.StorageClass}
 	}
+	if opts.WebsiteRedirectLocation != "" {
+		header["x-amz-website-redirect-location"] = []string{opts.WebsiteRedirectLocation}
```

Comment:
you should define this as a constant string just like `amzStorageClass`

Reply:
Done